### PR TITLE
Preloader Functionality Added

### LINKED ISSUE #17: Preloader Functionality
### Description
Add loading screen before the rendering of the application where it should show some decent animation with Hash Define written on it along with its logo.

You can use Lazy and suspense from react-router-dom to achieve this functionality.

### PR DESCRIPTION
**Pull request by shreya024**
I have added Preloader functionality.
Referring to issue number 
fixes #17

### CODE REVIEW

**Comment by DhairyaBahl** [comment]
@shreya024 Well done but That continuous scale on the logo is not looking as expected. How about just a logo in the middle of the screen. Remove the `HASH DEFINE` that you wrote below the image. and add a horizontal loading bar below the image. Hope you are getting my point.

**Comment by shreya024** [comment]
> @shreya024 Well done but That continuous scale on the logo is not looking as expected. How about just a logo in the middle of the screen. Remove the `HASH DEFINE` that you wrote below the image. and add a horizontal loading bar below the image. Hope you are getting my point.

Kindly check now .I have removed the text and added a horizontal loading bar but kept the animation.

File: src/Components/Preloader/Preloader.scss [hunk 1]
```diff
@@ -49,24 +57,24 @@
...
-    font-weight: bold;
+  .progress {
+    background: rgba(255, 255, 255, 0.1);
+    justify-content: flex-start;
+    border-radius: 100px;
+    align-items: center;
+    position: relative;
+    padding: 0 5px;
+    display: flex;
+    height: 40px;
```
Comment (vedant-jain03) [inline src/Components/Preloader/Preloader.scss:68]: Kindly make `height: 20px`

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]

**Comment by vedant-jain03** [comment]
Please uncommit `package-lock.json` file.

**Review by vedant-jain03 — APPROVED** [review]

**Comment by vedant-jain03** [comment]
Good work @shreya024

**Pull request merged**